modules/terraform/validate.go

```diff
@@ -1,10 +1,29 @@
 package terraform
 
 import (
+	go_test "testing"
+
 	"github.com/gruntwork-io/terratest/modules/testing"
 	"github.com/stretchr/testify/require"
 )
 
+// Automatically finds all folders specified in RootDir that contain .tf files and runs InitAndValidate in all of them
+// Excludes any folders specified in the ExcludeDirs slice
+func ValidateAllTerraformModules(t *go_test.T, opts ValidationOptions) error {
+	dirsToValidate, readErr := ReadModuleAndExampleSubDirs(opts)
+	if readErr != nil {
+		return readErr
+	}
+	for _, dir := range dirsToValidate {
+		t.Run(dir, func(t *go_test.T) {
+			t.Parallel()
+			tfOpts := &Options{TerraformDir: dir}
```

Comment:
Do you have to copy over any settings from the original `Options` the user passed in?

Reply:
I was originally thinking of support that - but I haven't given it deep thought yet. Right now we're only accepting the ValidationOptions. I figured most would follow our intended pattern of creating a single slim Go test that calls this method to run validate on every Terraform directory within their repo.

Reply:
I think that can be left as a future PR consideration based on usage in the wild.

Reply:
Cool, makes sense to me.